### ISSUE TITLE
Updated to work with latest bg.io version?

### ISSUE THREAD

**Issue by fladrif** [delucis/bgio-effects#571]
I'm trying to implement this with v0.50.0 of boardgame.io, but I'm seeing issues with the context object. Has bgio-effects' types and usage been updated to account for the new move and hook signatures?

Thanks!

**Comment by delucis** [comment]
No, I don’t think so, sorry! Haven’t had time to maintain this really. There is an open PR here: https://github.com/delucis/bgio-effects/pull/564

**Comment by fladrif** [comment]
oh nice thanks! I'll take a look at the PR. Any chance the PR gets merged?

**Comment by delucis** [comment]
I don’t really have the bandwidth to review, but if you can give it a review and leave comments, I could see if I remember how to release new versions of this repo 😅

**Comment by fladrif** [comment]
I'm inheriting the issue in this PR https://github.com/delucis/bgio-effects/pull/572
